create-publication-from-scopus/src/main/java/no/sikt/nva/scopus/ScopusHandler.java

```diff
@@ -86,6 +91,33 @@ private Reference generateReference(DocTp docTp) {
         return reference;
     }
 
+    private List<TitletextTp> getTitleText(DocTp docTp) {
+        return docTp.getItem().getItem().getBibrecord().getHead().getCitationTitle().getTitletext();
+    }
+
+    private String extractMainTitle(DocTp docTp) {
+        return getMainTitleTextTp(docTp)
+            .map(this::marshalMainTitleToXmlStringInOrderToPreserveUnderlyingSupAndInfTags)
+            .orElse(null);
+    }
+
+    private Optional<TitletextTp> getMainTitleTextTp(DocTp docTp) {
+        return getTitleText(docTp)
+            .stream()
+            .filter(this::isTitleOriginal)
+            .findFirst();
+    }
+
+    private boolean isTitleOriginal(TitletextTp titletextTp) {
+        return titletextTp.getOriginal().equals(YesnoAtt.Y);
+    }
+
+    private String marshalMainTitleToXmlStringInOrderToPreserveUnderlyingSupAndInfTags(TitletextTp contents) {
```

Comment:
marhalMainTitleToXmlPreservingUnderlyingStructure (we don't explicitly exclude anything, right?)

Reply:
Fixed